_posts/2015-10-07-code-talks-2015-retrospective.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "code.talks 2015 retrospective"
+date: "2015-10-07 09:00:00"
+icon: comments
+categories: conferences
+authors: ["Jens"]
+---
+
+For the fifth time about 1.500 developers gathered for the yearly `code.talks` conference in Hamburg:
+on September 29 and 30 also a dozen ePagees went to the Cinemaxx Hamburg Dammtor to enjoy two days fully packed with [lots of talks](https://www.codetalks.de/2015/programm) from 13 different tracks.
+Each track focussed on a specific theme like E-Commerce, IT Management, Startups, New Technology, Big Data, Mobile, DevOps, Scaling, Infrastructure, UX/Frontend, PHP, JavaScript and Java - the latter also being sponsored by ePages.
+Each hour talks from these tracks where held in parallel at eight movie theater rooms.
+Watching these talks on huge cinema screens while sitting in cozy chairs and eating popcorn with nachos is a very special experience!
+
+Due to the huge amount of different talks there was almost always at least one interesting session for everyone - more often than not it was a tough choice between two or more competing topics being presented at the same time.
+Between the talks there was time for meeting and networking with other visitors to exchange impressions and ideas.
+All the time there was more than enough excellent food and drinks provided; not only water, coffee and tea, but of course also [1337 mate](http://www.1337mate.com/) ;-)
```

Comment:
At all times there was more than enough excellent food and drinks provided; not only water, coffee and tea, but of course also [1337 mate](http://www.1337mate.com/) ;-)